tests/calibration_test.py

```diff
@@ -466,7 +497,7 @@ def test_calibration_inverse():
 
     # evaluate the inverse for a value outside the range
     with pytest.raises(CalibrationError):
-        cal1.inverse(-10.0)
+        cal1.inverse(-2e6)
     with pytest.raises(CalibrationError):
         cal1.inverse(2e6)
```

Comment:
Can you expand this to show that we can update the range of `cal1` and make the error disappear?